content/tech/2018-04-28-budgeting-a-conference.org

```diff
@@ -0,0 +1,170 @@
+Ruby on Ice 2018 and should help you as a conference organiser to put your own
+budget in perspective and give some transparency to you as a conference
+attendee. Note that this might not apply to large business conferences, but it
+worked for us as a community-conference.
+
+And before I dive in, a *big thank you* to Dajana, Bodo and Florian who helped
+us immensely in the run up to the conference by giving insight + feedback. <3
+
+* Retrospective
+
+We started planning for Ruby on Ice roughly one year ago - March / April 2017
+with a core team of three people. We had the backing of the Berlin-based
+Non-Profit [[https://rubyberlin.org/][Ruby Berlin e.V.]] that enabled us to have
+a legal entity for running the conference, selling tickets + sponsorships.
+
+A huge part of our planning time went into a huge Google Docs budget sheet. We
+adapted it from [[https://twitter.com/fannivuniconnu][@fannivuniconnu's]]
+version back from the eurucamp days and this was our guideline for all
+decisions.
+
+Our goal was to organise a community-run conference in the south of Germany with
+the backdrop of the Alps – inviting Rubyists from all over the world but also
+appealing to people from southern Germany, Austria and Switzerland who didn't
+want to travel to Berlin all the time.
+
+** Ticket sales vs. Sponsoring
+
+This was the first big head scratcher. Ruby Berlin e.V. is a Non-Profit. Sounds
+good, no? So at least we don't have to be profitable. Well. But we also should
+not operate at a 100% loss, as we want to keep the Verein healthy.
+
+I was involved in a lot of community-run conferences and used to ticket prices
+somewhere between *60-100€* for a two day event. Now – since I moved to Munich,
+I already expected it to be a bit more pricy, but the longer we shuffled around
+numbers the more we realised that we would have to raise the ticket prices to
+organise a conference with a balanced budget.
+
+Speaking of balance – most conferences rely on two revenue streams: /Ticket
+sales/ and /Sponsoring deals/. The first trap we probably stepped into was
+assuming that we could smoothly fill all the missing budget parts with
+sponsoring money.
+
+Well. We couldn't.
+
+*** On sponsoring »money«
+
+It's easy to think of Sponsoring »money« as a regular part of your budget or as
+some kind of benevolent donations. Except they're not. You _sell_ part of your
+stage and attention to a sponsor and you get paid for goods and services. Even
+though we're a Non-profit we have to pay taxes for those payments.
+
+A good baseline we adapted from grooming our budget with
+[[https://twitter.com/argorak][Florian]] was: 
+
+#+BEGIN_QUOTE
+Try to cover the base line of your cost (food, venue) with regular ticket sales and use the sponsoring money to afford nice things on top of that.
+#+END_QUOTE
+
+In the end we still evened out at a rough =50:50= ratio:
+
+[[file:https://img.bascht.com/2018-blog/rubyonice-chart-income.png]]
```

Comment:
That chart helps a lot to show the idea of "50/50" 👅 I don't see an alt text here.